src/mod_keystore.erl

```diff
@@ -22,8 +24,9 @@
               key_name/0,
               raw_key/0]).
 
--include("mongoose.hrl").
 -include("mod_keystore.hrl").
+-include("mongoose.hrl").
+-include("mongoose_config_spec.hrl").
 
 -define(DEFAULT_RAM_KEY_SIZE, 2048).
 -define(IOL2B(L), iolist_to_binary(L)).
```

Comment:
Below, in line 38:
`-type key_name() :: any().`
while we're validating the key name as `atom, non_empty`. Should we try to update this line to match better?

Reply:
We could do it. Not required, but nice to put `atom()` there.